Pontoon documentation: Update teams_projects.md page (#222)
mozilla-l10n/localizer-documentation: src/tools/pontoon/teams_projects.md

<!-- toc -->

> Pontoon organizes localizations by locale/project pairs. Pontoon's structure reflects this with separate pages for [Teams](#teams) (locales), [Projects](#projects) and [Localizations](#localization-page).

Reviewer: I'm not sure about this change:
* The existing page has a short explanation followed by the graph, now the graph is provided without context (the context is spread across the page later on).
* The current statement is not very clear (`Pontoon organizes localizations by locale/project pairs`)., specially when then you talk again about Localizations in the sentence after.

I think it makes sense to keep the current blurb (fixing the "you"), and accept that there will be some duplication.

The following diagram shows how these pages are organized and connected to each other:

![Team and project pages flow](../../assets/images/pontoon/teams_projects/page_flow.png)

## Teams

### Teams page

> The Teams page lists all locales that are enabled in Pontoon. It can be reached by clicking `Teams` from the navbar or by using the `/teams` URL (e.g. [pontoon.mozilla.org/teams](https://pontoon.mozilla.org/teams/)). Selecting a specific [Team page](#team-page) shows a list of all projects enabled for the requested locale. Choosing one of these projects leads to the [Localization page](#localization-page).

Reviewer: ```suggestion
The Teams page lists all locales that are enabled in Pontoon. It can be reached by clicking `Teams` from the navbar or by using the `/teams` URL (e.g. [pontoon.mozilla.org/teams](https://pontoon.mozilla.org/teams/)). Clicking on a specific locale will open the locale’s [Team page](#team-page).
```

I would move the "localization page" part into the team page section. Same for the Project page blurb.

### Team page

> You can access a specific team’s page by selecting it from the *Teams page* or by appending a locale code to the end of the Pontoon base URL with `/{LOCALE}` (e.g [pontoon.mozilla.org/it](https://pontoon.mozilla.org/it) for Italian).

Reviewer: ```suggestion
It’s possible to access a specific team’s page by selecting it from the *Teams page* or by appending a locale code to the end of the Pontoon base URL with `/{LOCALE}` (e.g [pontoon.mozilla.org/it](https://pontoon.mozilla.org/it) for Italian).
```

The page header contains linguistic information about the current locale (plural form, writing direction, etc.), plus an indication of the overall status of completion and statistics. The statistics displayed in the rightmost column are active links to access the Localization page for all projects. For example, select `MISSING` to see all missing strings across projects enabled for that locale in Pontoon.

![Header of team page for Italian](../../assets/images/pontoon/teams_projects/team_page_header.png)

> There are 5 tabs available to all roles, or 6 tabs (highlighted in red) for those with *admin* or *manager* roles:

Reviewer: ```suggestion
There are 5 tabs available to all roles, and 1 additional tab (highlighted in red) for those with *Administrator* or *Team manager* roles:
```

See http://localhost:3000/tools/pontoon/users.html#user-roles

Same note applies to the project page section.

* **Projects**: a list of all projects enabled in Pontoon for this locale.
* **Contributors**: a list of active contributors with their statistics, filterable by time (all time, last 12/6/3/1 months).
* **Insights**: contains data and trends presented in a graphical format about active users, time to review suggestions, review activity, and translation activity.
* **Bugs**: a list of open bugs for this locale, retrieved from [Bugzilla](https://bugzilla.mozilla.org/).
* **Info**: information about the team. Team managers can edit this by clicking on the Edit button.

> * **Permissions**: manage user permissions (see [this document](users.md#managing-permissions) for details).

Reviewer: ```suggestion
* **Permissions**: [manage user permissions](users.md#managing-permissions).
```

<!-- toc -->

> The [Teams page](https://pontoon.mozilla.org/teams/) lists all locales that are enabled in Pontoon. From here specific [Team pages](#team-page) can be accessed, which contains a list of all projects enabled for the requested locale. Selecting a project from the *Team page* leads to what is called the [Localization page](#localization-page).

Reviewer: ```suggestion
The Teams page, accessible using the `/teams` URL (e.g. [pontoon.mozilla.org/teams](https://pontoon.mozilla.org/teams)), lists all locales that are enabled in Pontoon. From here it’s possible to access a specific [Team page](#team-page), which contains a list of all projects enabled for the requested locale. Selecting a project from the *Team page* leads to what is called the [Localization page](#localization-page).
```

We need to use the same strategy we used in other docs, where pontoon.mozilla.org is an example, and this is a generic documentation.

> From each locale’s *Team page* it’s also possible to [Request a project](#requesting-a-project). Note that this is a request to add the locale to a project already available in Pontoon, not for requesting a brand new project.

Reviewer: ```suggestion
From each locale’s *Team page* it’s also possible to [Request a project](#requesting-a-project). Note that this is a request to add the locale to a project already available in Pontoon, it can’t be used for requesting a brand new project.
```

The alternative would be be "to add…, not to request".

> The [Projects page](https://pontoon.mozilla.org/teams/) lists all projects that are available in Pontoon. From here the list of locales that are enabled for the requested project can be accessed. This is called a [Project page](#project-page) in Pontoon. Access the [Localization page](#localization-page) by selecting a locale in the *Project page*.

Reviewer: ```suggestion
The Projects page, accessible using the `/projects` URL (e.g. [pontoon.mozilla.org/projects](https://pontoon.mozilla.org/projects)), lists all projects that are available in Pontoon. From here it’s possible to access a [Project page](#project-page), which shows all locales that are enabled for the requested project. Selecting a locale in the *Project page* leads to the [Localization page](#localization-page).
```

The Teams page lists all locales that are enabled in Pontoon. It can be reached by clicking `Teams` from the navbar or by using the `/teams` URL (e.g. [pontoon.mozilla.org/teams](https://pontoon.mozilla.org/teams/)). Clicking on a specific locale will open the locale's [Team page](#team-page).

### Team page

Access a specific team’s page by selecting it from the *Teams page* or by appending a locale code to the end of the Pontoon base URL with `/{LOCALE}` (e.g [pontoon.mozilla.org/it](https://pontoon.mozilla.org/it) for Italian). This page shows a list of all projects enabled for the locale. Clicking on one of these projects leads to the [Localization page](#localization-page).

The page header contains linguistic information about the current locale (plural form, writing direction, etc.), plus an indication of the overall status of completion and statistics. The statistics displayed in the rightmost column are active links to access the Localization page for all projects. For example, select `MISSING` to see all missing strings across projects enabled for that locale in Pontoon.

![Header of team page for Italian](../../assets/images/pontoon/teams_projects/team_page_header.png)

> There are 5 tabs available to all roles, and 1 additional tab for those with *Administrator* or *Team manager* roles (highlighted in red above):

Author: Yeah, seems like this doesn't need to specifically be called out. Will remove.

There are 6 tabs available (highlighted in red):
* **Resources**: files available in the project.
* **Tags**: a list of tags defined for this project. This tab will be visible only in projects where tags are enabled. For more information about tags, see [this page](search_filters.md#tags).
* **Contributors**: a list of active contributors with their statistics, filterable by time. Unlike the tab available in the *Team page*, this only includes contributions to localization of the project for the current locale.
* **Insights**: contains data and trends presented in a graphical format about review activity and translation activity.
* **Project info**: information about the project.
* **Team info**: same content as the *Info* tab in the *Team page*.

In addition to the *Localization page* for each locale/project pair, there is also a special Localization page that allows access to all resources for all projects enabled for a specific locale. This can be accessed from the `/{LOCALE}/all-projects/all-resources` URL (e.g. [pontoon.mozilla.org/it/all-projects/all-resources](https://pontoon.mozilla.org/it/all-projects/all-resources) for Italian).

## Insights graphs

> The *Insights tab*, accessible from either the *Project*, *Team*, or *Localization page*, displays data and trends on contributor and localization activity in a graphical format. Depending on the page, the following insights may be shown:

Reviewer: Instead of repeating `(Only appears on Team page)`, I would split those 3 graphs in a separate list.

Intro for the first list `The following insights are displayed in all different pages:`, for the new list `These insights are displayed only in the Team page:`.

![Request a project](../../assets/images/pontoon/teams_projects/request_locale.png)

> Click on `REQUEST NEW LANGUAGE`, select the language to add and then click `REQUEST NEW LANGUAGE`.

Reviewer: For a follow-up, we should consider if this should be called "REQUEST NEW LOCALE", since language is not really correct.

Reviewer: Filed https://github.com/mozilla/pontoon/issues/2870
